packages/cli/src/commands/create-github-app/index.ts

```diff
@@ -47,3 +49,14 @@ integrations:
         - $include: ${fileName}`),
   );
 };
+
+async function verifyGithubOrg(org: string): Promise<void> {
+  const response = await fetch(
+    `https://api.github.com/orgs/${encodeURIComponent(org)}`,
+  );
+  if (!response.ok) {
+    throw new Error(
+      `GitHub organization '${org}' does not exist. Please verify the name and try again.`,
```

Comment:
Perhaps check that we're getting the expected error code here? Thinking we'd let this through if there's a network issue, perhaps with a warning

Reply:
I tweaked it slightly, PTAL